devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -16,210 +16,309 @@
+        fingerprint mfr:"0086", prod:"0103", model:"006F", deviceJoinName: "Aeon Labs Nano Dimmer"
+    }
+
+    simulator {
+        status "on":  "command: 2603, payload: FF"
+        status "off": "command: 2603, payload: 00"
+        status "09%": "command: 2603, payload: 09"
+        status "10%": "command: 2603, payload: 0A"
+        status "33%": "command: 2603, payload: 21"
+        status "66%": "command: 2603, payload: 42"
+        status "99%": "command: 2603, payload: 63"
+
+        for (int i = 0; i <= 10000; i += 1000) {
+            status "power  ${i} W": new physicalgraph.zwave.Zwave().meterV3.meterReport(
+                scaledMeterValue: i, precision: 3, meterType: 4, scale: 2, size: 4).incomingMessage()
+        }
+        for (int i = 0; i <= 100; i += 10) {
+            status "energy  ${i} kWh": new physicalgraph.zwave.Zwave().meterV3.meterReport(
+                scaledMeterValue: i, precision: 3, meterType: 0, scale: 0, size: 4).incomingMessage()
+        }
+
+        ["FF", "00", "09", "0A", "21", "42", "63"].each { val ->
+            reply "2001$val,delay 100,2602": "command: 2603, payload: $val"
+        }
+    }
+
+    tiles(scale: 2) {
+        multiAttributeTile(name:"switch", type: "lighting", width: 6, height: 4, canChangeIcon: true){
+            tileAttribute ("device.switch", key: "PRIMARY_CONTROL") {
+                attributeState "on", label:'${name}', action:"switch.off", icon:"st.switches.switch.on", backgroundColor:"#00a0dc", nextState:"turningOff"
+                attributeState "off", label:'${name}', action:"switch.on", icon:"st.switches.switch.off", backgroundColor:"#ffffff", nextState:"turningOn"
+                attributeState "turningOn", label:'${name}', action:"switch.off", icon:"st.switches.switch.on", backgroundColor:"#00a0dc", nextState:"turningOff"
+                attributeState "turningOff", label:'${name}', action:"switch.on", icon:"st.switches.switch.off", backgroundColor:"#ffffff", nextState:"turningOn"
+            }
+            tileAttribute ("device.level", key: "SLIDER_CONTROL") {
+                attributeState "level", action:"switch level.setLevel"
+            }
+        }
+        valueTile("power", "device.power", width: 2, height: 2) {
+            state "default", label:'${currentValue} W'
+        }
+        valueTile("energy", "device.energy", width: 2, height: 2) {
+            state "default", label:'${currentValue} kWh'
+        }
+        standardTile("reset", "device.energy", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", label:'reset kWh', action:"reset"
+        }
+        standardTile("refresh", "device.switch", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
+        }
+    }
+
+    main(["switch","power","energy"])
+    details(["switch", "power", "energy", "refresh", "configure", "reset"])
```

Comment:
This is the list of tiles, it should not have "configure" here as that is not a tile defined above.